src/raft/raft.rs

```diff
@@ -1219,66 +1266,28 @@ impl<T: Storage> Raft<T> {
         maybe_commit: &mut bool,
         more_to_send: &mut Option<Message>,
     ) {
-        if !self.prs.contains_key(&m.get_from()) {
+        if self.get_prs().get(&m.get_from()).is_none() {
```

Comment:
Why not use `contains`?

Reply:
`FlatMap` doesn't have method `contains`.

Reply:
How about `contains_key`?